_inc/client/discussion/comments.jsx

```diff
@@ -19,27 +19,12 @@ import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
 
 class CommentsComponent extends React.Component {
-	/**
-	 * If markdown module is inactive and this is toggling markdown for comments on, activate module.
-	 * If markdown for posts is off and this is toggling markdown for comments off, deactivate module.
-	 *
-	 * @param {string} module	the module slug.
-	 * @returns {*}             the updated value
-	 */
-	updateFormStateByMarkdown = module => {
-		if ( !! this.props.getSettingCurrentValue( 'wpcom_publish_posts_with_markdown', module ) ) {
-			return this.props.updateFormStateModuleOption( module, 'wpcom_publish_comments_with_markdown' );
-		}
-		return this.props.updateFormStateModuleOption( module, 'wpcom_publish_comments_with_markdown', true );
-	};
-
 	render() {
 		const foundComments = this.props.isModuleFound( 'comments' ),
 			foundGravatar = this.props.isModuleFound( 'gravatar-hovercards' ),
-			foundMarkdown = this.props.isModuleFound( 'markdown' ),
 			foundCommentLikes = this.props.isModuleFound( 'comment-likes' );
 
-		if ( ! foundComments && ! foundGravatar && ! foundMarkdown && ! foundCommentLikes ) {
+		if ( ! foundComments && ! foundGravatar && && ! foundCommentLikes ) {
```

Comment:
This is causing an error on `yarn build` due to the double `&&`.

Reply:
D'oh, amended the commit but forgot to force-push.